Snakefile

```diff
@@ -109,7 +109,7 @@ rule prepare_vehicle_charging_demand:
     output:
         "results/_resources/ts_load_electricity_vehicles.csv"
     shell:
```

Comment:
You could add logging. Have a look at the other rules to see how.

Reply:
Yes, nice! Added in [5a3af6a](https://github.com/rl-institut/oemof-B3/pull/196/commits/5a3af6a958e5f916105f571c83319682306f65a9)